lib/trader.js

```diff
@@ -317,6 +317,15 @@ Trader.prototype.triggerSuspend = function triggerSuspend (customerId, triggerId
     .then(r => r.body)
 }
 
+Trader.prototype.smsReceipt = function smsReceipt (data, customerId) {
+  return this.request({
+    path: `/customer/${customerId}/smsreceipt`,
+    body: { data },
```

Comment:
Isn't `data` already an object?

Reply:
You're accessing this with `req.body.data` on server-side, due to this. Changing this has little to no impact, just pointed it out due to other functions in the trader module sending the object as is, without encapsulating on a `data` field